### ISSUE TITLE
RSS / Torrent Download Manger

### ISSUE THREAD

**Issue by hi5kba** [BiglySoftware/BiglyBT#420]
i just recently found out about BigleyBt.
At current i use Vuze (for Bakabt) and Utorrent for any public RSS series.

one flaw i found with Vuze is that it doesn't allow you define How a RSS feed is handled so you have to Manually check and start new torrents in the feed, which is why i run Utorrent as it allows me to preset the download location, Tags, Piority per feed separate from the Dinfined directory in the main settings.

is it Possible to get a similar Feature in this?

reference -- [utorrent](http://help.utorrent.com/customer/en/portal/articles/163700-how-do-i-import-torrents-from-rss-feeds-)
if you would like screen shots of what i mean i can do so!!

**Comment by Havokdan** [comment]
Have good plugins, do you try: https://plugins.biglybt.com

**Comment by parg** [comment]
You can associate a Tag with a Subscription (right-click on it in the Side Bar for example) - you can then associate an initial download location with the Tag.

**Comment by ferdnyc** [comment]
Yeah, I have all of those things set automatically on all of my RSS Feed Scanner downloads, mostly via the application of appropriate Tags. Download location, check. Priority, check (well, I'm not sure what you mean by that exactly, but you can set max up/down speed and target ratio per tag.) Tags themselves are set by RSS Feed Scanner, which can also filter the feed's contents using regular expression matches.

There's little you _can't_ do with Tags, TBH. One feature I like is the ability to create tags that auto-apply _themselves_ to any torrents that contain matching tracker hostnames. So I can have my torrents from private trackers tagged automatically, without having to lift a finger. The tag is configured with a set of sharing parameters appropriate to meet that site's rules.
